src/resources/elements/address-link/address-link.html

```diff
@@ -0,0 +1,9 @@
+<template>
```

Comment:
Where is this component used?

Reply:
It is being used in the transactions grid. Brandon created the component to be reusable for any address link